xl_auth/templates/users/view.html

```diff
@@ -35,11 +35,24 @@ <h1>{{ _('View User \'%(email)s\'', email=user.email) }}</h1>
                     {{ user.created_by.full_name }}
                 </a>
             </dd>
+
+            <dt>{{ _('Cataloging admin for') }}:</dt>
```

Comment:
I think this is a bit funky;

![screen shot 2017-12-07 at 1 53 08 pm](https://user-images.githubusercontent.com/786326/33716977-79902c66-db58-11e7-8c39-3cd51a280066.png)

"Katalogiseringsadmin", and again "Cataloging admin for". Would be prettier like this, methinks:

![screen shot 2017-12-07 at 2 09 27 pm](https://user-images.githubusercontent.com/786326/33717024-b5969a06-db58-11e7-92f6-bc23950249ae.png)

How I made it work:

```jinja
            <dt>{{ _('Cataloguing Administrator') }}:</dt>
            <dd>
                {% if user.is_cataloging_admin %}
                    {{ _('Yes; for ') }}
                    {% for permission in user.get_cataloging_admin_permissions()
                            | sort(attribute='collection.code') %}
                        <a href="{{ url_for('collection.view',
                                            collection_code=permission.collection.code) }}">{{
                            permission.collection.code }}</a>{{
                            _(' and ') if loop.revindex == 2 else (', ' if not loop.last) }}
                    {% endfor %}
                {% else %}
                    {{ _('No') }}
                {% endif %}
            </dd>
```

Reply:
@mxtthias Another UX remark.. I propose we change 
`<dt>{{ _('Administrator') }}:</dt>  ->  <dt>{{ _('System Administrator') }}:</dt>`, to differentiate the two; being a "cataloging administrator" might (incorrectly) imply you're also an "administrator".

What do you think?